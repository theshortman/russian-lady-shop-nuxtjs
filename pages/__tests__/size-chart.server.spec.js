/**
 * @jest-environment node
 */

import { renderToString } from '@vue/server-test-utils'
import SizeChartPage from '../size-chart.vue'

test('renders correctly size chart page', async () => {
  const str = await renderToString(SizeChartPage)
  
  expect(str).toMatchSnapshot()
})