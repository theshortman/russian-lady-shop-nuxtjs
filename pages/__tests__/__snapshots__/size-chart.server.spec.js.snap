// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly size chart page 1`] = `
"<div data-server-rendered=\\"true\\"><h1>Таблица размеров</h1> <div><p class=\\"fw-bold\\">Уважаемые клиенты!</p> <p>Размеры, указанные в каталоге являются российскими.</p> <p><span class=\\"text-danger\\">Как измерить свои размеры для выбора одежды.</span> <br>
      Производите замеры в нижнем белье. Не напрягайтесь. Прижимайте
      измерительную ленту вплотную к телу.
    </p> <p>
      1. Обхват груди: измеряется горизонтально по наиболее выступающим точкам
      груди.
    </p> <p>
      2. Обхват бедер: измеряется горизонтально по наиболее выступающим точкам
      бедер и ягодиц. При измерении поставьте ноги вместе, измеряйте бедра в
      большей их части, примерно на 15-20 см ниже талии.
    </p> <p><span class=\\"fw-bold\\">Размерный код равен обхвату груди деленному на два.</span><br>
      Пример: если обхват груди равен 112 см, это соответствует 56-му размеру.
    </p> <p>
      Все размеры указаны в сантиметрах.<br>
      Измерения типовых фигур женщин рост 164, 170, 176.
    </p> <div class=\\"table-responsive text-nowrap\\"><table class=\\"table table-bordered\\"><thead><tr><th scope=\\"col\\">Размер</th> <th scope=\\"col\\">48</th> <th scope=\\"col\\">50</th> <th scope=\\"col\\">52</th> <th scope=\\"col\\">54</th> <th scope=\\"col\\">56</th> <th scope=\\"col\\">58</th> <th scope=\\"col\\">60</th></tr></thead> <tbody><tr><th scope=\\"row\\">Обхват груди</th> <td>96</td> <td>100</td> <td>104</td> <td>108</td> <td>112</td> <td>116</td> <td>120</td></tr> <tr><th scope=\\"row\\">Обхват бедер</th> <td>106</td> <td>110</td> <td>114</td> <td>118</td> <td>122</td> <td>126</td> <td>130</td></tr></tbody></table></div></div></div>"
`;
